keps/sig-storage/1682-csi-driver-skip-permission/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: Skip Volume Ownership Change
+kep-number: 1682
+authors:
+  - "@huffmanca"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+status: implementable
+creation-date: 2020-04-27
+reviewers:
+  - "@gnuified"
+  - "@msau42"
+  - "@xing-yang"
+approvers:
+  - "@saad-ali"
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
```

Comment:
Can you fill out this section too?

Reply:
I originally had these values uncommented; however, it failed verification as `stage` wasn't an expected value in the kep. I noticed another submitted KEP had left these commented out, so I did the same to get it to pass the verification test.

Reply:
That's strange. @johnbelamaric do you know if that's intentional or the template should be updated to remove stage?

For now can you leave out the stage, but fill out the rest? Here's another example: https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1412-immutable-secrets-and-configmaps/kep.yaml

Reply:
I've kept `stage` out, and the rest is filled in. It looks like the test is still failing; however, if I run `hack/verify-kep-metadata.sh` locally it passes without issue.

Note that running the test locally still fails if `stage` is included.

Reply:
```
../../keps/sig-storage/1682-csi-driver-skip-permission/kep.yaml has an error: "yaml: unmarshal errors:\n  line 33: field rollback-supported not found in type keps.Proposal"
```

Maybe the version of the verify script on your local machine is not up to date?

Reply:
After doing a rebase this passes locally with `stage` included as well.

Apologies for the issues.